extensions/identity-hub/src/main/java/org/eclipse/dataspaceconnector/identityhub/api/IdentityHubController.java

```diff
@@ -0,0 +1,74 @@
+/*
+ *  Copyright (c) 2022 Microsoft Corporation
+ *
+ *  This program and the accompanying materials are made available under the
+ *  terms of the Apache License, Version 2.0 which is available at
+ *  https://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  SPDX-License-Identifier: Apache-2.0
+ *
+ *  Contributors:
+ *       Microsoft Corporation - initial API and implementation
+ *
+ */
+
+package org.eclipse.dataspaceconnector.identityhub.api;
+
+import io.swagger.v3.oas.annotations.Operation;
+import io.swagger.v3.oas.annotations.tags.Tag;
+import jakarta.ws.rs.Consumes;
+import jakarta.ws.rs.POST;
+import jakarta.ws.rs.Path;
+import jakarta.ws.rs.Produces;
+import org.eclipse.dataspaceconnector.identityhub.dtos.MessageRequestObject;
+import org.eclipse.dataspaceconnector.identityhub.dtos.MessageResponseObject;
+import org.eclipse.dataspaceconnector.identityhub.dtos.RequestObject;
+import org.eclipse.dataspaceconnector.identityhub.dtos.RequestStatus;
+import org.eclipse.dataspaceconnector.identityhub.dtos.ResponseObject;
+import org.eclipse.dataspaceconnector.identityhub.dtos.WebNodeInterfaces;
+import org.eclipse.dataspaceconnector.identityhub.processor.MessageProcessorRegistry;
+
+import java.util.stream.Collectors;
+
+import static org.eclipse.dataspaceconnector.identityhub.dtos.WebNodeInterfaceMethod.fromName;
+
+/**
+ * Identity Hub controller, exposing a <a href="https://identity.foundation/decentralized-web-node/spec">Decentralized Web Node</a> compatible endpoint.
+ *
+ * See {@link WebNodeInterfaces} for a list of currently supported DWN interfaces.
+ */
+@Tag(name = "IdentityHub")
+@Produces({"application/json"})
```

Comment:
```suggestion
@Produces({ "application/json" })
```
Reformat all files with EDC codestyle.xml

Reply:
What is the value of that? Reformatting with EDC codestyle.xml brings all kinds of changes that actually decrease readability of the code (like rest assured indentations for `given().when().then()`). I think we should focus on readability and not 100% compliance with EDC's code style.

Reply:
This is in this repo's style guide https://github.com/agera-edc/IdentityHub/blob/main/CONTRIBUTING.md#adhere-to-coding-style-guide.

The purpose is to reduce diffs, if everyone uses the same codestyle.xml